src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -67,20 +70,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   /** Stores a new {@link Message}. */
   @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException
+  {
 
     UserService userService = UserServiceFactory.getUserService();
-    if (!userService.isUserLoggedIn()) {
+    if (!userService.isUserLoggedIn())
+    {
       response.sendRedirect("/index.html");
       return;
     }
 
     String user = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+    String recipient = request.getParameter("recipient"); //added recipient variable
```

Comment:
The comment `added recipient variable` is not necessary because it is clear that the code is getting a recipient from the request. Use comments to explain complex parts of the code and refrain from commenting on code that is self explanatory.